docs/assembling_twir.md

```diff
@@ -6,9 +6,7 @@ This documents the general process for assembling each of the sections.
 
 ## Updates from the Rust Community
 
-[@nellshamrell](https://github.com/nellshamrell) currently covers this section, but would welcome help from additional editors!
-
-(Thank you to [@cdmistman](https://github.com/cdmistman) for covering this section so long! We look forward to when you can return!)
+The full editoral staff works together to cover this section, but the primary contributors are [@extrawurst](https://github.com/extrawurst), [@ericseppanen](https://github.com/ericseppanen), and [@cdmistman](https://github.com/cdmistman). 
```

Comment:
Would it make sense to note that @nellshamrell provides lead editorial guidance for the overarching process?

Reply:
yes! That absolutely does. Want to add something in there, or want me to?

Reply:
I'll defer to your judgment :)